### PR TITLE
[W5.11R][T09-B4] Daniel Ng Sze Yao

### PR DESCRIPTION
**Pull request by ghost**
- Implemented the ability for the address book to sort the people in it by name alphabetically by adding a "sort" command.
- Updated necessary java files and also added a new functional class called SortCommand.java
- Performed necessary tests using JUnit

Acknowledgments given to @traceurgan for inspiration :)

### CODE REVIEW

File: doc/UserGuide.md [hunk 1]
```diff
@@ -81,6 +84,10 @@ Examples:
   `view 1`<br>
   Views the 1st person in the results of the `find` command.
 
+## Sorting all entries : 'Sort'
+Sort all people in the address book by name alphabetically.<br>
```
Comment (jeffryhartanto) [inline doc/UserGuide.md:88]: Good job for updating the UserGuide, but you can give an example here.

File: src/seedu/addressbook/commands/SortCommand.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.Person;
+
+import java.util.List;
+
+public class SortCommand extends Command {
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/commands/SortCommand.java:7]: No header comment.

File: src/seedu/addressbook/commands/HelpCommand.java [hunk 3]
```diff
@@ -17,6 +16,7 @@
             + "\n" + FindCommand.MESSAGE_USAGE
             + "\n" + ListCommand.MESSAGE_USAGE
             + "\n" + ViewCommand.MESSAGE_USAGE
+            + "\n" + SortCommand.MESSAGE_USAGE
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/commands/HelpCommand.java:19]: Good job for updating the help command.

**Comment by jeffryhartanto** [comment]
@HEARTOFAL1ON Some comments added. Please close the PR after reading comments.

**Comment by ghost** [comment]
Thank you for your review

File: src/seedu/addressbook/commands/SortCommand.java (hunk 2 above)
Comment (ghost) [reply src/seedu/addressbook/commands/SortCommand.java:7]: OK I will take note in future and remember that every non-trivial class and non-trivial method inside a class requires a header comment whenever possible :)

**Pull request closed without merging**